NCLCoreClasses/UISystem.h

```diff
@@ -19,6 +19,11 @@ namespace NCL {
 		constexpr int SUSPISION_INDICATOR_SLOT = 7;
 		constexpr int CROSS = 8;
 		constexpr int ALARM = 9;
+		constexpr int NOTICERIGHT = 10;
+		constexpr int NOTICELEFT = 11;
+		constexpr int NOTICETOP = 12;
+		constexpr int NOTICEBOT = 13;
```

Comment:
At this point you are sort of reverse engineering an Enum, and while both work, maybe turn this into an Enum instead